cognite/client/data_classes/geospatial.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Dict, Union, List
+from typing import Any, Dict, List, Union
```

Comment:
isort?

Reply:
probably, but I use whatever configuration we have in the repo :/ idk why geospatial gets formatted, I thought it would have been formatted when you merge PRs.

Reply:
@tuanng-cognite @vincent-cognite could you stamp this pr and #900 ?